hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/RootEndpoint.java

```diff
@@ -61,6 +62,9 @@ public Response get()
         throw e;
       }
 
+      OzoneVolume volume = getVolume();
+      response.setOwner(new S3Owner(volume.getName(), volume.getName()));
```

Comment:
```suggestion
      response.setOwner(new S3Owner(0, volume.getOwner()));
```

The first argument of S3Owner is ID, which is a 64-character encoded unique user name. We dont' support it in Ozone. https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-identifiers.html

@ivandika3 any ideas about the owner id?

Reply:
I think we can make it simple. See how MinIO does it: https://github.com/minio/minio/issues/4553
Ceph S3 does not compute a user name hash either.